File: ExternalTransmittal-deployDev_030317/ViewController/src/login/AfterLogin.java
package login;


import com.utils.SessionClass;

import java.text.SimpleDateFormat;

import java.util.Calendar;

import javax.faces.event.PhaseEvent;

import oracle.adf.view.rich.component.rich.input.RichInputText;


public class AfterLogin {
    private RichInputText username;
    private RichInputText password;
    public SessionClass ses = new SessionClass();
    public AfterLogin() {
    }

    public void setUsername(RichInputText username) {
        this.username = username;
    }

    public RichInputText getUsername() {
        return username;
    }

    public void setPassword(RichInputText password) {
        this.password = password;
    }

    public RichInputText getPassword() {
        return password;
    }

    public void GetSession(PhaseEvent phaseEvent) {
        // Add event code here...
        this.username.setValue(ses.getSessionParameter("username"));
        this.password.setValue(ses.getSessionParameter("password"));
        System.out.println("ExternalTransmittal:Login:AfterLogin:Username: " + this.username.getValue());
      //  System.out.println("ExternalTransmittal:Login:AfterLogin:Password: " + this.password.getValue());
        
      Calendar cal = Calendar.getInstance();
      SimpleDateFormat sdf = new SimpleDateFormat("HH:mm:ss");
      System.out.println("ExternalTransmittal:Login:AfterLogin:time: " + sdf.format(cal.getTime())); 
    }
}//Menggunakan lempar parameter tanpa cookie 

/*
//Menggunakan cookie dev version tanpa decript
package login;


import com.utils.SessionClass;

import javax.faces.event.PhaseEvent;

import oracle.adf.view.rich.component.rich.input.RichInputText;


public class AfterLogin {
    private RichInputText username;
    private RichInputText password;
    public SessionClass ses = new SessionClass();
    public AfterLogin() {
    }

    public void setUsername(RichInputText username) {
        this.username = username;
    }

    public RichInputText getUsername() {
        return username;
    }

    public void setPassword(RichInputText password) {
        this.password = password;
    }

    public RichInputText getPassword() {
        return password;
    }

    public void GetSession(PhaseEvent phaseEvent) {
        // Add event code here...
//        ses.setSessionParameter("username", ses.getParameter("j_username"));
//        ses.setSessionParameter("password", ses.getParameter("j_password"));
//        System.out.println("usernamePara set session : " + ses.getParameter("j_username"));
//        System.out.println("passParam set session: " + ses.getParameter("j_password"));
        
        this.username.setValue(ses.getSessionParameter("username"));
        this.password.setValue(ses.getSessionParameter("password"));
        System.out.println("usernamePara setValue: " + this.username.getValue());      
        System.out.println("passParam setValue: " + this.password.getValue());
    }
}
*/

File: ExternalTransmittal-deployDev_030317/ViewController/public_html/AfterLogin.js
$( document ).ready(function() {
    var username = document.getElementById('j_username::content').value;
    var password = document.getElementById('j_password::content').value;
    //var username = document.getElementById('j_username::content').value;
    //var password = document.getElementById('j_password::content').value;
    
//    window.location="http://kponwjap005:16300/urm/login/j_security_check?j_username=" + 
//        username + "&j_password=" + password;
 //   window.location="http://kponwjis013.pertamina.com:16300/urm/login/j_security_check?j_username=" + 
 //       username + "&j_password=" + password;
//    window.location="http://kponwjis013.pertamina.com:16300/urm/login/j_security_check?j_username=" + 
//        username + "&j_password=" + password;
}); //Menggunakan lempar parameter tanpa cookie 

/*
$( document ).ready(function() {
 
    var username = document.getElementById('j_username::content').value;
    var password = document.getElementById('j_password::content').value;
    
//nanda - 2702
//modified by nanda - 250215 - throw URM login in post form//
    document.loginURM.j_username.value = username;
    document.loginURM.j_password.value = password;
    document.forms["loginURM"].submit();

});//Menggunakan cookie dev version tanpa decript*/